<?xml version="1.0" encoding="UTF-8"?><testrun name="CoffeeMakerTest" project="CoffeeMaker" tests="6" started="6" failures="1" errors="0" ignored="0">
  <testsuite name="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.005">
    <testcase name="testAddRecipe1" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.0"/>
    <testcase name="testAddRecipe2" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.0"/>
    <testcase name="testAddRecipe3" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.0"/>
    <testcase name="testAddRecipe4" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.004">
      <failure>junit.framework.AssertionFailedError&#13;
	at junit.framework.Assert.fail(Assert.java:55)&#13;
	at junit.framework.Assert.assertTrue(Assert.java:22)&#13;
	at junit.framework.Assert.assertFalse(Assert.java:39)&#13;
	at junit.framework.Assert.assertFalse(Assert.java:47)&#13;
	at junit.framework.TestCase.assertFalse(TestCase.java:219)&#13;
	at edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest.testAddRecipe4(CoffeeMakerTest.java:92)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testEditRecipe1" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.001"/>
    <testcase name="testDeleteRecipe1" classname="edu.towson.cis.cosc442.project4.coffeemaker.CoffeeMakerTest" time="0.0"/>
  </testsuite>
</testrun>
